### ISSUE TITLE
No module named block_picker_multiple

### ISSUE THREAD

**Issue by blueForestIcarus** [mcedit/mcedit2#213]
I have been trying to get mcedit2 running on Arch Linux. Python spits out this error when I attempt to start the program.

```
[ERROR][main.py:190]:Unhandled Exception: 
	No module named block_picker_multiple
Traceback (most recent call last):
  File "/home/bficarus/.bin/mcedit2/ENV/bin/mcedit2", line 9, in <module>  
    load_entry_point('mcedit2', 'console_scripts', 'mcedit2')()
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/main.py", line 207, in main  
    app = startup()
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/util/profiler.py", line 73, in _wrapper (self is a Profiler)
    return func(*a, **kw)
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/main.py", line 179, in startup  
    from mcedit2.editorapp import MCEditApp
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/editorapp.py", line 14, in <module>  
    from mcedit2 import plugins
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/plugins.py", line 11, in <module>  
    from mcedit2.editortools import generate
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/editortools/generate.py", line 19, in <module>  
    from mcedit2.widgets.layout import Column
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/widgets/__init__.py", line 7, in <module>  
    import blockpicker
  File "/home/bficarus/.bin/mcedit2/ENV/test/src/mcedit2/widgets/blockpicker.py", line 12, in <module>  
    from mcedit2.ui.widgets.block_picker_multiple import Ui_blockPickerMultiple
ImportError: No module named block_picker_multiple
```

I checked the ui.widgets directory and found it to contain no python files, only ".ui" files. Don't you have to compile .ui files to python files to import them like that? Is that something setup.py is supposed to do?

**Comment by codewarrior0** [comment]
It should generate the py files when you run from source. If it doesn't, run `python src/mcedit2/util/gen_ui.py` to generate them.

**Comment by blueForestIcarus** [comment]
I will try that on a fresh install. 
(I already managed to get it to work by running "for f in `find . -regex ".*\.ui"`; do pyside-uic $f > ${f::-2}'py'; done" which should have done the same thing. No problems as of yet.)

**Comment by QuantumWarpCode** [comment]
I'm having the same problem on windows and the gen_ui.py thing isn't working.

**Comment by QuantumWarpCode** [comment]
I tried this in a batchfile and it appears to work but no files are generated:

for %%a in (G:\Programs\Minecraft\MCEdit2\src\mcedit2\ui\*.ui) do G:\Programs\Minecraft\MCEdit2\ENV\Scripts\pyside-uic.exe "%%a"

**Comment by codewarrior0** [comment]
Should be fixed in 98fd8f5cc24a9751a01974f91b5e9dec87847161
